### PR TITLE
https://issues.redhat.com/browse/ACM-6906--removing content that is re…

### PR DESCRIPTION
**Pull request by swopebe**
…dundant, reducing files, adding links to support matrixes that may need to change

### CODE REVIEW

File: clusters/about/requirements.adoc [hunk 1]
```diff
@@ -1,8 +0,0 @@
-[#requirements-and-recommendations]
-= Requirements and recommendations
```
Comment (swopebe) [inline clusters/about/requirements.adoc:2]: this file is silly :-) Moving this info to install overview.

File: install/requirements.adoc [hunk 2]
```diff
@@ -1,33 +0,0 @@
-[#requirements-and-recommendations]
-= Requirements and recommendations
```
Comment (swopebe) [inline install/requirements.adoc:2]: https://redhat-internal.slack.com/archives/CUHCCV19P/p1695847285014559

Comment (swopebe) [reply install/requirements.adoc:2]: we already say this

**Review by cadawson — CHANGES_REQUESTED** [review]
Looks good. Trouble is we will have to find out how to update the link on the product page to the latest versions of the support matrix if we want to use that link. Or we could use a conref for the link directly to the support matrix. Or we could hardcode it. We can figure something out.

**Pull request merged**